### ISSUE TITLE
ubiquity: Swap partition doesn't appear in the partitionning message

### ISSUE THREAD

**Issue by InfoLibre** [linuxmint/mint20-beta#20]
Linux Mint MATE 20.
I chose "Effacer le disque et installer Linux Mint" :

![Capture du 2020-06-14 13-49-24](https://user-images.githubusercontent.com/3955499/84592426-ebc8c000-ae45-11ea-9c4e-42b425c36513.png)

After, in the confirmation message, "swap" doesn't appears :

![Installation_9](https://user-images.githubusercontent.com/3955499/84592439-04d17100-ae46-11ea-8ead-f9c89dee427b.png)

**Comment by enigma131** [comment]
In the same way, I'm still on Usb live so no install at the moment, no hibernation I presume ? This is a Ubuntu missing feature since 18.04, was present before... Debian bases strangely made it... and is appreciate on laptops. Auto suspend on RAM is OK only if you have fresh battery and if you do it with old battery a night long you will loose your datas for sure.

**Comment by InfoLibre** [comment]
They were this bug in Mint MATE 19 : https://github.com/linuxmint/linuxmint/issues/169
But I can't reproduce it on Mint MATE 20, screensaver seems not to work in VirtualBox.

**Comment by mirostauder** [comment]
Using custom setup,
I noticed the swap partition was listed for formatting despite having a pre-configured swap available

**Comment by mtwebster** [comment]
This I think some of this may be due to the fact that a swap *file* is used now instead of a partition (for a default installation) since ubuntu 18.04.

**Comment by InfoLibre** [comment]
Indeed. In fact, it is the UEFI WIN95 FAT32 partition name that misses :

![Capture du 2020-06-15 21-53-10](https://user-images.githubusercontent.com/3955499/84699995-c4abe480-af52-11ea-9947-7c2b43bc397c.png)

(and I don't understand why / is in an extended partition and not a primary partition)

**Comment by clefebvre** [comment]
Yes, Ubuntu (and Mint) moved to a swapfile a while ago.

**Comment by InfoLibre** [comment]
Yes, but "... de type FAT32" should be written instead of "... de type"

**Comment by clefebvre** [comment]
ah sorry, you're right.
